po/R-es.po

```diff
@@ -218,23 +221,23 @@ msgid ""
 "The actual schedule may vary slightly depending on the topics and exercises "
 "chosen by the instructor."
 msgstr "El horario real puede variar ligeramente dependiendo de los temas y "
-"ejercicios elegidos por el instructor."
+"ejercicios elegidos por la instructora."
```

Comment:
:-)

Reply:
Using the feminine as default is part of the agreements the Carpentries Spanish-speaking community made.  I'm only applying those.  :-)